scripts/functions/requirements/fedora_dnf

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+
+requirements_fedora_lib_installed()
+{
+  dnf list installed "$1" >/dev/null 2>&1 || return $?
+}
+
+requirements_fedora_lib_available()
+{
+  dnf info "$1" >/dev/null 2>&1 || return $?
+}
+
+requirements_fedora_libs_install()
+{
+  __rvm_try_sudo dnf install -y "$@" || return $?
+}
+
+requirements_fedora_libs_remove()
+{
+  __rvm_try_sudo dnf remove -y "$@" || return $?
+}
+
+requirements_fedora_check_binary()
+{
+  {
+    __rvm_which update-alternatives >/dev/null &&
+    update-alternatives --display "$1" >/dev/null 2>&1
+  } ||
+  __rvm_which "$1" >/dev/null || return $?
+}
+
+
+requirements_fedora_define_openssl()
+{
+  case "$1" in
+    (ruby-2.3*|ruby-2.2*|ruby-2.1*|ruby-2.0*|ruby-1.9*|ruby-1.8*)
+      if
+        __rvm_version_compare "${_system_version}" -ge 26
```

Comment:
most likely not needed because of https://github.com/rvm/rvm/pull/4198/files#diff-3b57f99cafe3ce5bc01f41f9a3573c0eR4 ... unless I missed something?